src/components/questions/AudioRecorder.tsx

```diff
@@ -217,24 +218,25 @@ const AudioRecorder = (props: {
             disabled={isDisabled}
             activeOpacity={0.5}
             onPress={onStopPlay}
+            accessible={true}
             accessibilityLabel={'Spraakopname starten vanaf het begin knop'}
-            accessibilityHint={'Ga terug naar begin opname'}
           >
             <Icon name="backward" size={48} style={isDisabled ? styles.disabled : styles.icon} />
```

Comment:
Of overal knop gebruiken of overal niet

Reply:
Zou hier ook van maken: 'Ga terug naar begin van de opname'